buildcommon.xml

```diff
@@ -1,179 +1,170 @@
-<?xml version="1.0" ?>

```

Comment:
Why the whole file has changed?

Reply:
No idea.
The spacing is the same, and it is an UTF-8 CRLF file on master and on that branch.
But GitHub see some whitespace change on each line nonetheless.

Reply:
It seems this had already happened previously as suggested by [this comment](https://github.com/nhibernate/NHibernate-Caches/pull/74#issue-308386205), but a quick look in the history has not shown me when.